.github/workflows/ci-sbt.yml

```diff
@@ -0,0 +1,49 @@
+# Find full documentation here https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
+name: CI
+
+on:
+  pull_request:
+
+  # Manual invocation.
+  workflow_dispatch:
+
+  push:
+    branches:
+      - main
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+
+    # See https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
+    permissions:
+      # required by guardian/actions-assume-aws-role
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+
+      # Setup AWS credentials to enable uploading to S3 for Riff-Raff.
+      # See https://github.com/guardian/actions-assume-aws-role
+      - uses: guardian/actions-assume-aws-role@v1
+        with:
+          awsRoleToAssume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+
+      # Change this if using Java 11.
+      # Configuring caching is also recommended.
+      # See https://github.com/actions/setup-java
+      - name: Setup Java 8
+        uses: actions/setup-java@v2
+        with:
+          java-version: '8'
+          distribution: 'adopt'
+
+      # 1. Seed the build number with last number from TeamCity.
+      #    Update `LAST_TEAMCITY_BUILD` as needed or remove entirely if changing Riff-Raff project name.
+      # 2. Execute `script/ci`, a script that will compile, run tests etc.
+      #    Ensure this file exists in your repository and has executable permissions (`chmod u+x script/ci`).
+      # See https://github.com/github/scripts-to-rule-them-all
+      - name: Run script/ci
+        run: |
+          LAST_TEAMCITY_BUILD=0
```

Comment:
You'll want to set this to non-zero, and pause TeamCity, if you want to have continuous numbers with TeamCity.

Reply:
I think we're currently overwriting previous builds with this being set to 0. Not a huge deal as we're never going to deploy such old versions, but better to seed GHA from the last TC build.

Reply:
@akash1810 hey yep, at the moment we're on 8 here, and I was content to overwrite (a couple of very old) things as didn't want to block TC deploys. But yeah, we should set this once I've got the okay to merge.

Reply:
If negative numbers work then maybe there's no need to overwrite things? No harm though, I can't see us wanting to deploy anything that old.